### ISSUE TITLE
Windows setup - clone

### ISSUE THREAD

**Issue by jonmaiga** [loda-lang/loda-cpp#71]
Followed the instructions as per https://loda-lang.org/install/

```
Press return to download the default programs repository:
[https://github.com/loda-lang/loda-programs.git]
fatal: Too many arguments.
```

If it helps, my user directory has a space.

**Comment by ckrause** [comment]
Yep, that makes sense. Should be easy to fix.

**Comment by ckrause** [comment]
fixed in v21.12.1
